addons/ai/stringtable.xml

```diff
@@ -404,5 +404,17 @@
             <Chinese>武器瞄準</Chinese>
             <Italian>Obiettivo dell'arma</Italian>
         </Key>
+        <Key ID="STR_ZEN_AI_FireWeapon">
+            <English>Weapon Fire</English>
```

Comment:
Why is this backwards? Should be "Fire Weapon".

Reply:
If it prevents AI from firing their weapon, maybe just call it shooting.

Reply:
Changed to "Fire Weapon" and sorted alphabetically